README.md

```diff
@@ -99,19 +89,14 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
```

Comment:
[OPTIONAL] 
- [ ] please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1: